draft-ietf-httpbis-sfbis.md

```diff
@@ -1042,7 +1049,7 @@ The size of most types defined by Structured Fields is not limited; as a result,
 It is possible for parties with the ability to inject new HTTP fields to change the meaning
 of a Structured Field. In some circumstances, this will cause parsing to fail, but it is not possible to reliably fail in all such circumstances.
 
-The Display String type conveys a Unicode string without any form of sanitization. Applications using these values need to perform their own checks on their content; for example, they might contain escape sequences, or NUL. Mitigation strategies include escaping untrusted content before displaying it.
+The Display String type can convey all possible Unicode code points without any form of sanitization; for example, they might contain unassigned code points, surrogates, control points (including NUL), or non-characters. Therefore, applications consuming Display Strings need to consider strategies such as filtering or escaping untrusted content before displaying it. See also {{UNICODE-SECURITY}} and {{?I-D.draft-bray-unichars}}.
```

Comment:
If we use UTF-8, can it really contain surrogates?

Reply:
Ah, good point.

Reply:
Considering how often one sees ill-formed UTF-16 data mindlessly converted into UTF-8, I would be inclined to leave surrogates in the list.

Reply:
I think that Mark's text is weak, but probably the best we can hope to come up with in that regard.  Yes, they will occur because rules for handling noncharacters or unpaired surrogates are special and often missed.  They will also occur because attacks using these characters are potentially interesting if handling is poor.

Some additional comments:

* Unassigned codepoints are not the same class of problem as these others.
* Do you mean "non-characters" or "noncharacters"?  I understand why the hyphen appears, but it might not be correct.

Reply:
Changed to 'noncharacters'